tests/unit/components/buttons/HdRadioButton.spec.js

```diff
@@ -24,24 +26,22 @@ describe('HdRadioButton', () => {
         mobileIcon,
       },
     });
-
-    wrapper.setMethods({ select: mockedSelect });
   });
 
   test('the component is rendered', () => {
     expect(wrapper.html()).toMatchSnapshot();
+    expect(wrapper.find(ICON_SELECTOR).attributes().style).toBe(BACKGROUND_STYLE);
```

Comment:
I believe that this one can be on its own test: "Shows the provided icon" or something like this.
And to be a bit flexible you can use `stringContaining` instead of `toBe` in case we add more styles.

Reply:
Also just be aware that this https://github.com/homeday-de/homeday-blocks/blob/develop/src/components/buttons/HdRadioButton.vue#L23-L26 can also be tested the way you're doing. I do recommend adding it as well 😄

Reply:
Done 👍